main.tf

```
@@ -53,7 +53,7 @@ resource "aws_dynamodb_table" "default" {
   hash_key         = "${var.hash_key}"
   range_key        = "${var.range_key}"
   stream_enabled   = "${var.enable_streams}"
-  stream_view_type = "${var.stream_view_type}"
+  stream_view_type = "${var.enable_streams ? var.stream_view_type : ""}"
```

```suggestion
  stream_view_type = "${var.enable_streams == "true" ? var.stream_view_type : ""}"
```

since `enable_streams` is a string.

Also, the default value is already an empty string
https://github.com/cloudposse/terraform-aws-dynamodb/blob/master/variables.tf#L72

so it should work without the change

---

@aknysh 

> Also, the default value is already an empty string
Oh, I missed this one. In my local copy, I was adding some default value. Thanks for pointing out.